packages/@sanity/migrate/src/mutations/asserters.ts

```diff
@@ -22,3 +23,32 @@ export function isTransaction(mutation: unknown): mutation is Transaction {
     mutation.type === 'transaction'
   )
 }
+
+export function isOperation(value: unknown): value is Operation {
+  return (
+    value !== null &&
+    typeof value === 'object' &&
```

Comment:
should we be worried about arrays in here?

Reply:
Not at this point, but I'll look into narrowing down the `value` argument here.